components/brave_shields/ios/browser/brave_shields_utils.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_SHIELDS_IOS_BROWSER_BRAVE_SHIELDS_UTILS_H_
+#define BRAVE_COMPONENTS_BRAVE_SHIELDS_IOS_BROWSER_BRAVE_SHIELDS_UTILS_H_
+
+#ifdef __cplusplus
+#include "brave/components/brave_shields/ios/common/brave_shields_panel.mojom.objc.h"
```

Comment:
the relevant settings should be moved out of `brave_shields_panel`. I guess for now call the new file `brave_shields_modes.mojom`?